tests/test_utils.py

```diff
@@ -33,3 +38,8 @@ def test_split_name(self):
 
     def test_format_twitter_handle(self):
         assert format_twitter_handle("testusername") == "@testusername"
+
+    def test_null_abort(self, test_client):
+        with current_app.test_request_context('/'):
+            with pytest.raises(expected_exception=BadRequest):
+                abort_null('\x00')
```

Comment:
You need a second test to verify (a) it doesn't blow up given clean input, and (b) it returns the input.